backend/src/apiserver/storage/job_store.go

```diff
@@ -344,55 +344,52 @@ func (s *JobStore) EnableJob(id string, enabled bool) error {
 	return nil
 }
 
-func (s *JobStore) UpdateJob(swf *util.ScheduledWorkflow) error {
+func (s *JobStore) UpdateJob(j *model.Job) error {
 	now := s.time.Now().Unix()
-	parameters, err := swf.ParametersAsString()
-	if err != nil {
-		return err
-	}
 
-	sql, args, err := sq.
+	sqlStatement, args, err := sq.
```

Comment:
A thought on this: if we only allow "" to be updated, can we have less columns in this sql query? Otherwise, if we want the job_store's method to be generic and can be used later when we allow more columns to be updated, can we have all the columns here? I didn't check very carefully but it seems that the columns listed here is a set of the all the columns but more than the updatable field allowed so far.

Reply:
this same method is used by `ReportScheduledWorkflowResource` though, which updates the SWF and I didn't want to inadvertently affect that method.